packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -39,7 +40,30 @@ class HelpGroup {
         process.stdout.write('\n                  Made with ♥️  by the webpack team \n');
     }
 
-    outputVersion() {
+    outputVersion(externalPkg) {
+        const commandsUsed = (commands) => {
+            return process.argv.filter((val) => commands.find(({ name }) => name === val));
+        };
+
+        if (externalPkg && commandsUsed(commands).length === 1) {
+            try {
+                if (commands.filter((cmd) => cmd.name === externalPkg.name)) {
+                    const { name, version } = require(`@webpack-cli/${externalPkg.name}/package.json`);
+                    process.stdout.write(`\n${name} ${version}`);
```

Comment:
You can take this statement outside the `if/else` block.

Reply:
this refactor will be like below -

```
const { name, version } = commandsUsed().includes(externalPkg.name) ? require(`@webpack-cli/${externalPkg.name}/package.json`) : require(`${externalPkg.name}/package.json`);

process.stdout.write(`\n${name} ${version}`);
```
as done here  [HelpGroup#L7](https://github.com/webpack/webpack-cli/blob/848d80ee6a67fffae8d41d78f3097ef3cd1f4614/packages/webpack-cli/lib/groups/HelpGroup.js#L7).

it will be too long for a single line. Should I do this?